book/02-git-basics/sections/viewing-history.asc

```diff
@@ -264,7 +265,7 @@ In <<limit_options>> we'll list these and a few other common options for your re
 [cols="2,4",options="header"]
 |================================
 | Option                | Description
-| `-(n)`                | Show only the last n commits
+| `-(n)`                | Show only the last n commits (really a shortcut for `-n (n)`, so `-5` and `-n 5` are equivalent)
```

Comment:
Oh, actually it looks like it's not a problem. I ran a test build on Atlas, and here's how it comes out:

![image](https://cloud.githubusercontent.com/assets/39902/17646635/61e0dca6-6187-11e6-870b-3fd56f80b570.png)

So never mind!